packages/vite/src/client/client.ts

```diff
@@ -126,6 +126,7 @@ function cleanUrl(pathname: string): string {
 }
 
 let isFirstUpdate = true
+const invalidLinkTags = new Set<HTMLLinkElement>()
```

Comment:
I feel "invalid" slightly different from the meaning here. Maybe "outdated" or "stale"?